src/gluonts/model/forecast_generator.py

```diff
@@ -82,6 +82,14 @@ def make_distribution_forecast(distr, *args, **kwargs) -> Forecast:
     raise NotImplementedError
 
 
+def make_predictions(prediction_net, inputs: dict):
+    # MXNet predictors only support positional arguments
+    if prediction_net.__class__.__module__.startswith("gluonts.mx"):
```

Comment:
I couldn't find a more elegant way to use different logic for MXNet and PyTorch models :/

I tried `@singledispatch`, but that doesn't work for subclasses (i.e., we'd need to define it for all subclasses of `pl.LightningModule` in GluonTS, and same for MXNet).